deps/noise/noise.c

```diff
@@ -109,9 +111,10 @@ static unsigned char PERM[] = {
     128, 195,  78,  66, 215,  61, 156, 180
 };
 
+
 void seed(unsigned int x) {
     srand(x);
-    for (int i = 0; i < 256; i++) {
+    for (int i = 0; i < 255; i++) {
```

Comment:
Why does the bound for this loop change to 255 instead of 256?

Reply:
Honestly, it was initially a mistake when I was trying to revert code I was playing around with.  I left if after looking at it because iterating past 255 is overwritten by the next block of code making an exact copy of PERM[0] through PERM[255] into PERM + 256.